CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Collaborative Changelog
+
+## [NEXT-TO-DO] - 2023-04-23
+- [ ] add styling for button/input focus/hover states
+- [ ] fix scrolling on homepage
+- [ ] add live number of subs (Joe needs to do since I don’t have access to the store)
```

Comment:
such a good idea! i'll probably switch from Reform to Beehiiv so I can actually email people